install-dependencies.sh

```diff
@@ -296,6 +297,7 @@ elif [ "$ID" = "fedora" ]; then
     # Disable C extensions
     pip3 install scylla-driver --install-option="--no-murmur3" --install-option="--no-libev" --install-option="--no-cython"
     pip3 install traceback-with-variables
+    pip3 install scylla-api-client
```

Comment:
I see it's under fedora section, what about ubuntu?

Reply:
Why we maintain only Fedora dependencies is, because of frozen toolchain and relocatable packages.
Since we introduced frozen toolchain and relocatable packages, we build everything in Fedora Docker container, not on target distribution (so we build in Fedora even we want to build Ubuntu packages).
Since then, we don't need to officially support building Scylla in other distributions anymore (the docker container can run every distribution, so we don't need to support them directly).

So install-dependencies.sh has entries for other distributions, but we don't really maintain them.
Especially packaging thing is designed to build in our build infrastructure so it work just on the docker container.